rs/cli/src/cli.rs

```diff
@@ -120,6 +122,16 @@ pub(crate) enum Commands {
         /// all subnets
         subnet_ids: Vec<PrincipalId>,
     },
+    /// Dump full registry as json
+    DumpRegistry {
```

Comment:
We should think in the future about putting all registry-related ops into a separate subcommand. I don't see an immediate need for this since we have only one function for the registry, which is "dump". But we can revisit that later.